_includes/notice/notice-de.md

```diff
@@ -1,3 +1,3 @@
 <p>Diese Übersetzung zur Verfügung gestellt von StrongLoop / IBM.</p>
 
-Dieses Dokument kann im Vergleich zur englischen Dokumentation veraltet sein. Aktuelle Updates finden Sie in der <a href='{{ page.url | replace: page.lang, "en" }}'>englischen Dokumentation</a>.
+Dieses Dokument kann im Vergleich zur englischen Dokumentation veraltet sein. Aktuelle Updates finden Sie in der <a href='{{ page.url | replace: "de/", "en/" }}'>englischen Dokumentation</a>.
```

Comment:
I think this would be a bit safer

```suggestion
Dieses Dokument kann im Vergleich zur englischen Dokumentation veraltet sein. Aktuelle Updates finden Sie in der <a href='{{ page.url | replace: "/de/", "/en/" }}'>englischen Dokumentation</a>.
```